src/web/src/components/Header.vue

```diff
@@ -113,11 +110,36 @@ export default {
   data() {
     return {
       semesterOptions: [],
+      followDevice: (this.$cookies.get('darkMode') === null),
     };
   },
   methods: {
     toggle_style() {
+      if (this.$cookies.get("darkMode") === null) {
+        this.$bvToast.toast(
+          `Double click moon icon to follow device's color scheme.`, {
+            title: 'Color Scheme Changed',
+            autoHideDelay: 2000,
+            noHoverPause: true,
+            variant: "info",
+        });
+      }
```

Comment:
wyt about extracting these toasts into named functions? could make `toggle_style()` and `toggle_default()` a little easier to read

Reply:
Totally agree.  These `toast` calls will be wrapped as functions and moved to the bottom of `methods:` on the next commit.

Reply:
btw, no need to worry about showering commits in for prs. they'll get squashed as we merge into master.